src/pages/data-reporting/reports-api/v2.0/index.mdx

```diff
@@ -350,6 +366,8 @@ curl "https://api.livechatinc.com/reports/chats/ratings?group=2&date_from=2012-0
 
 <Text>
 
+This method is indirectly used by one of our Sample Apps, the <a href="https://github.com/livechat/progress-app" target="_blank">Progress</a> app. Check it out and experiment with an already existing app.
```

Comment:
I'm not sure if anybody will see this sentence here below all the tables and code blocks

Reply:
I'm thinking that maybe it would be more practical to only have the Sample app section. If we change something in endpoints/app, we'll need to update every single mention for each method. And it seems like to Sample app section has good visibility and is put contextually. The same for the other app.

Reply:
When you put it this way, I totally agree😉